wfc_dylib/src/lib.rs

```diff
@@ -501,6 +544,10 @@ pub unsafe extern "C" fn wfc_observe(
     }
 }
 
+// TODO(yan): Import/Export convert indices to positions and back, set bits by
+// one instead of copying memory, etc... Should we expose the underlying
+// representation?
```

Comment:
Or rather, make the API more granular in the future, having functions like `wfc_world_state_slot_set(handle, slot_x, slot_y, slot_z, module_id, true/false)`